llvm/lib/Target/AMDGPU/VOP1Instructions.td

```diff
@@ -635,8 +635,8 @@ def VOPProfile_Base_CVT_F32_F8_OpSel : VOPProfile<[f32, i32, untyped, untyped]>
   let Src1VOP3DPP = Src1RC64;
 }
 
-let SubtargetPredicate = isGFX12Plus, mayRaiseFPException = 0,
-    SchedRW = [WriteFloatCvt] in {
+let SubtargetPredicate = isGFX12Plus, OtherPredicates = [HasFP8ConversionInsts],
```

Comment:
Why not just SubtargetPredicate = HasFP8ConversionInsts?

Reply:
I think these instructions have gfx940 and gfx12+ versions. Not sure whether it will affect gfx940 if we only use  SubtargetPredicate = HasFP8ConversionInsts.

Reply:
I see... That sound right.